pages/1.12/installing/evaluation/gcp/index.md

```diff
@@ -99,23 +99,38 @@ Password: `deleteme`
   - ```public-agent-loadbalancer``` - Specifies the URL of your Public routable services.
 
   ```hcl
+  provider "google" {
+    version = "~> 1.0"
+  }
+
+  # Used to determine your public IP for forwarding rules
+  data "http" "whatismyip" {
+    url = "http://whatismyip.akamai.com/"
+  }
+
   variable "dcos_install_mode" {
     description = "specifies which type of command to execute. Options: install or upgrade"
     default = "install"
   }
 
   module "dcos" {
     source = "dcos-terraform/dcos/gcp"
+    version = "~> 0.1"
```

Comment:
this should be `~> 0.1.0` to follow only patches as minor version could have breaking changes and api changes